bindata/network/ovn-kubernetes/self-hosted/ovnkube-master.yaml

```diff
@@ -381,6 +381,15 @@ spec:
                 {{ else }}
                 ipsec=false
                 {{ end }}
+                if [ "${ipsec}" == "true" ]; then
```

Comment:
@mccv1r0 you could combine this with the `set nb_global` below and do it all in one?

Reply:
It has to be 

```sudo ovn-nbctl set nb_global . ipsec=${ipsec} options:ipsec_encapsulation=${encapsulation}```

where ${encapsulation} defaults to false and is set true only for IBMCloud